apps/backoffice-v2/src/domains/business-reports/fetchers.ts

```diff
@@ -88,3 +89,39 @@ export const fetchBusinessReports = async ({
 
   return handleZodError(error, filter);
 };
+
+export const createBusinessReport = async ({
+  websiteUrl,
+  operatingCountry,
+  companyName,
+  businessCorrelationId,
+  reportType,
+}:
+  | {
+      websiteUrl: string;
+      operatingCountry?: string;
+      reportType: TBusinessReportType;
+      companyName: string;
+    }
+  | {
+      websiteUrl: string;
+      operatingCountry?: string;
+      reportType: TBusinessReportType;
+      businessCorrelationId: string;
+    }) => {
+  const [businessReport, error] = await apiClient({
+    endpoint: `business-reports`,
+    method: Method.POST,
+    schema: z.undefined(),
```

Comment:
why `schema: z.undefined()`?

Reply:
The endpoint returns `void`. Using `z.void` would result in a union of `void | undefined`